### ISSUE TITLE
Missing parameter `digest`

### ISSUE THREAD

**Issue by AlCalzone** [ioBroker/ioBroker.js-controller#853]
https://github.com/ioBroker/ioBroker.js-controller/blob/acb2ce7c022e77b10086dd472ae6f3abea9c2ab6/lib/password.js#L62

See [documentation](https://nodejs.org/dist/latest-v8.x/docs/api/crypto.html#crypto_crypto_pbkdf2_password_salt_iterations_keylen_digest_callback)

```
crypto.pbkdf2(password, salt, iterations, keylen, digest, callback)
```

Version | Changes
-- | --
v8.0.0 | The digest parameter is always required now.
v6.0.0 | Calling this function without passing the digest parameter is deprecated now and will emit a warning.

**Comment by AlCalzone** [comment]
According to the Node.js 4 docs, 'sha1' used to be the default.

**Comment by AlCalzone** [comment]
Fix is up in #860
